### PR TITLE
Travis CI

### PR DESCRIPTION
**Pull request by justinmc**
https://travis-ci.org/Autodesk/molecular-design-toolkit

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+  - "2.7"
+addons:
+  apt:
+    packages:
+      - python-zmq
```
Comment (avirshup) [inline .travis.yml]: Just out of curiosity - is there any particular reason for you were installing ZMQ?

**Comment by avirshup** [comment]
Thanks for getting this set up, Justin!

I'm going to play around a bit with the "install" portion to see what I can get running.
In the end, we'll have 4 deployment targets with each release:
 - PyPI
 - [Docker images](https://hub.docker.com/r/autodesk/moldesign/) (stored at `docker.io/autodesk/moldesign:[IMAGENAME]-[MDT_VERSION]`)
 - [Conda](https://anaconda.org/moldesign)
 - binder.io demo server (see http://mybinder.org/repo/avirshup/mdt-gallery-test/ and https://github.com/avirshup/)

Because [the documentation](https://autodesk.github.io/molecular-design-toolkit/) isn't really being written as the same time as our releases, it's probably OK to keep that as a manual deployment (a deployment script would definitely be nice though)

**Comment by avirshup** [comment]
Travis is up and running tests as of 58161ac - but exiting with 1, which makes it look like our builds are failing. Not sure if this is the correct behavior or not; I'd ideally like to get stats on the number of passing/failing tests rather than just a blanket pass/fail

**Comment by avirshup** [comment]
This is now working - the build is failing because some of the tests are legitimately failing. (tried to trigger a push to PyPI by adding the 0.7.4alpha0 tag. It didn't work ... because the build is failing.)

In any case, I think this is ready to merge - I have a couple of upcoming PRs to fix the current test failures, and we won't be able to deal with deployment until that's done.

@justinmc, what do you think?

**Comment by justinmc** [comment]
Awesome!  This is a huge step forward for the project.  I agree with that plan.  Merge this now while it's failing, and then put in new PRs to fix/improve it later 👍

**Comment by avirshup** [comment]
> This is a huge step forward for the project

Agreed - thanks for getting this rolling!

**Pull request merged**